docker-app/qfieldcloud/core/admin.py

```diff
@@ -727,6 +728,24 @@ def has_delete_permission(self, request, obj):
         return False
 
 
+class IsFinalizedDeltaJobFilter(admin.SimpleListFilter):
+    title = _("finalized delta job")
+    parameter_name = "finalized"
+
+    def lookups(self, request, model_admin):
+        return (
+            ("finalized", _("finalized")),
+            ("not finalized", _("not finalized")),
+        )
+
+    def queryset(self, request, queryset):
+        q = Q(last_status="pending") | Q(last_status="started")
+        if self.value() == "not finalized":
+            return queryset.filter(q)
+        else:
+            return queryset.filter(~q)
```

Comment:
@why-not-try-calmer hey, actually this is never checking if the selected filter is ALL and always filter for either `finilized` or `not finalized`.

Reply:
What is `ALL`? `ALL` == "the user is not applying the filter at all" ? Or is it the disjunction of all statuses?